src/nixos/machines/morph/services/distributedBuilds.nix

```diff
@@ -6,19 +6,20 @@
 
 let
 	inherit (lib) mkIf;
-in {
-	# Authorize TSVETAN
-		# users.extraUsers.builder.openssh.authorizedKeys.keys = [
-		# 	"ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIF18QG9oqeeq/lQc5QDJl3hz5D4Q9bhiHFTRLJN4KSZb" # TSVETAN
-		# ];
+in  mkIf config.nix.distributedBuilds {
+	# Authorized Builders
+		users.extraUsers.builder.openssh.authorizedKeys.keys = [
+			"ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDhD5Fel4xaocToIQay3IkytHGaK93cDN52ww2Bw5Nj+" # IGNUCIUS (Builder)
```

Comment:
Figure out why is morph refusing connections from nix daemon from ignucius for distributed builds